GliaWidgets/Sources/View/Chat/Message/Content/Text/ChatTextContentView.swift

```diff
@@ -89,19 +94,52 @@ class ChatTextContentView: BaseView {
     }
 
     private func setText(_ text: String?) {
-        if text == nil || text?.isEmpty == true {
+        guard let text, !text.isEmpty else {
             textView.removeFromSuperview()
-        } else {
-            if textView.superview == nil {
-                contentView.addSubview(textView)
-                textView.translatesAutoresizingMaskIntoConstraints = false
-                var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
-                constraints += textView.layoutInSuperview(insets: kTextInsets)
-            }
-            textView.text = text
+            return
+        }
+
+        if textView.superview == nil {
+            contentView.addSubview(textView)
+            textView.translatesAutoresizingMaskIntoConstraints = false
+            var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
+            constraints += textView.layoutInSuperview(insets: kTextInsets)
         }
+        textView.text = text
+
         textView.accessibilityIdentifier = text
     }
+
+    private func setAttributedText(_ text: NSAttributedString?) {
+        guard let text, !text.string.isEmpty else {
+            textView.removeFromSuperview()
```

Comment:
This seems like a quite big side effect of sending an empty/nil string though. Like imagine if setting `label.text = ""` would mea that the label gets removed from the superview. I guess it's more expected that this gets checked before, so 

```
if text.string.isEmpty {
    text.removeFromSuperview()
} else {
    text.setAttributedText(...)
}
```

Reply:
I analyzed this file more and it turns out that the textView is a private thing, it's an internal implementation of the content view that the caller is not aware of. Then I guess there's not much to do